apps/explorer/src/lib/server/latest-block.server.ts

```diff
@@ -11,16 +11,21 @@ const QB = IDX.QueryBuilder.from(IS)
 
 export const fetchLatestBlock = createServerFn({ method: 'GET' }).handler(
 	async () => {
-		const config = getWagmiConfig()
-		const chainId = getChainId(config)
+		try {
+			const config = getWagmiConfig()
+			const chainId = getChainId(config)
 
-		const result = await QB.selectFrom('blocks')
-			.select('num')
-			.where('chain', '=', chainId)
-			.orderBy('num', 'desc')
-			.limit(1)
-			.executeTakeFirstOrThrow()
+			const result = await QB.selectFrom('blocks')
+				.select('num')
+				.where('chain', '=', chainId)
+				.orderBy('num', 'desc')
+				.limit(1)
+				.executeTakeFirst()
```

Comment:
Do you still want `executeTakeFirstOrThrow` so `catch` can be hit?

```suggestion
				.executeTakeFirstOrThrow()
```